### ISSUE TITLE
Installation of Makie stuck

### ISSUE THREAD

**Issue by vectorml** [JuliaPlots/Makie.jl#781]
I am trying to install Makie through VS Studio using ] add Makie. Its stuck on the following line:
```
[4607b0f0] + SuiteSparse
Building ModernGL → `C:\Users\julia_user\.julia\packages\ModernGL\rVuW2\deps\build.log` 
Building FFTW ────→ `C:\Users\julia_use\.julia\packages\FFTW\DMUbN\deps\build.log`
Building GLMakie ─→ `C:\Users\julia_user\.julia\packages\GLMakie\Pway1\deps\build.log`
```

**Comment by vectorml** [comment]
I tried the following steps to rectify the issue: 

1. Closed VS Studio
2. Restarted VS STudio and opened the Julia terminal. 
3. removed Makie using ] `rm Makie`
4. Deleted `Makie` folder .julia\packages
5. Installed Makie again using `] add Makie`

I found the procedure on https://discourse.julialang.org/t/julia-hanging-pausing-indefinitely-on-package-build-of-mbedtls-and-others/11316

Can anyone explain the reason for the issue.
